app/models/framework/definition/RM3767.rb

```diff
@@ -11,7 +11,6 @@ class Invoice < EntryData
 
         field 'Lot Number', :string, exports_to: 'LotNumber', presence: true, lot_in_agreement: true
         field 'Customer URN', :integer, exports_to: 'CustomerURN', urn: true
-        # field 'Tyre Specification', :string, exports_to: 'ProductCode', presence: true
```

Comment:
As I noted on our phone call, these were fields that existed in the MISO export but were missing from the Excel templates themselves.  I think these can be removed however.